ops/storage-ddls/cassandra-setup.cql

```diff
@@ -30,30 +30,26 @@ CREATE TABLE sitesettings(
     cogvisionsvctoken text,
     cogtextsvctoken text,
     insertion_time timestamp,
-    PRIMARY KEY (id, sitename)
+    PRIMARY KEY (sitename)
```

Comment:
Given that the size of this table will be zero or one, we can always just add `ALLOW FILTERING` here :P